tests/notus/test_notus.py

```diff
@@ -0,0 +1,414 @@
+# -*- coding: utf-8 -*-
+# Copyright (C) 2014-2020 Greenbone Networks GmbH
+#
+# SPDX-License-Identifier: AGPL-3.0-or-later
+#
+# This program is free software: you can redistribute it and/or modify
+# it under the terms of the GNU Affero General Public License as
+# published by the Free Software Foundation, either version 3 of the
+# License, or (at your option) any later version.
+#
+# This program is distributed in the hope that it will be useful,
+# but WITHOUT ANY WARRANTY; without even the implied warranty of
+# MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the
+# GNU Affero General Public License for more details.
+#
+# You should have received a copy of the GNU Affero General Public License
+# along with this program. If not, see <http://www.gnu.org/licenses/>.
+
+import logging
+import unittest
+
+from csv import DictReader
+from pathlib import Path, PurePath
+from collections import OrderedDict
+from unittest.mock import patch, MagicMock
+
+from ospd_openvas.notus.metadata import (
+    NotusMetadataHandler,
+    EXPECTED_FIELD_NAMES_LIST,
+    METADATA_DIRECTORY_NAME,
+)
+from ospd_openvas.errors import OspdOpenvasError
+from tests.helper import assert_called_once, assert_called
```

Comment:
I am not sure if these helpers are still necessary if we are using Python >= 3.7 now :-)